piccolo/apps/migrations/auto/migration_manager.py

```diff
@@ -472,7 +472,14 @@ async def _run_alter_columns(self, backwards: bool = False):
                 if (old_column_class is not None) and (
                     column_class is not None
                 ):
-                    if old_column_class != column_class:
+                    base_colum_class = params.get("base_column").__class__
```

Comment:
There's a typo here - `base_colum_class` -> `base_column_class`.

Reply:
Thanks. I will change that